eq-author/src/graphql/fragments/question-confirmation.graphql

```diff
@@ -3,12 +3,34 @@ fragment QuestionConfirmation on QuestionConfirmation {
   title
   displayName
   positive {
+    id
     label
     description
+    validationErrorInfo {
+      id
+      errors {
+        id
+        type
+        field
+        errorCode
+      }
+      totalCount
+    }
   }
   negative {
+    id
     label
     description
+    validationErrorInfo {
+      id
+      errors {
+        id
+        type
+        field
+        errorCode
+      }
+      totalCount
+    }
```

Comment:
Can we import the fragments here to compact this query a little further?

Reply:
Done.